pages/wg/working-groups.md

```diff
@@ -75,3 +75,10 @@ the US-RSE slack or contact the
 
 Visit the [Outreach Working Group page]({{ site.baseurl }}/wg/outreach/) for
 more details and a catalog of work products.
+
+
+<h3><a href="{{ site.baseurl }}/wg/rse-in-natl-labs/">RSE Empowerment in National Labs</a></h3>
+
+The RSE Empowerment in National Labs working group, conceptualized in 2022, is for those who are interested in working towards the empowerment of RSEs in national labs and other similar organizations. RSEs from national labs are an important part of the RSE community in the United States. The goal of this working group is to understand the specific challenges faced by RSEs in this ecosystem, advocate the goals and mission of US-RSE, assimilate resources that help us educate and train RSEs, increase RSE awareness across the entire spectrum and create a community and sense of belonging among the RSEs at National Labs.  If you would like to join the conversation, please join us on the US-RSE Slack or contact us on <a href="mailto:wg-national-labs-empowerment@us-rse.org">national lab empowerment working group list</a>. 
```

Comment:
```suggestion
The RSE Empowerment in National Labs working group, conceptualized in 2022, is for those who are interested in working towards the empowerment of RSEs in national labs and other similar organizations.
RSEs from national labs are an important part of the RSE community in the United States.
The goal of this working group is to understand the specific challenges faced by RSEs in this ecosystem, advocate the goals and mission of US-RSE, assimilate resources that help us educate and train RSEs, increase RSE awareness across the entire spectrum, and create a community and sense of belonging among the RSEs at National Labs.
If you would like to join the conversation, please join us on the US-RSE Slack or contact us on <a href="mailto:wg-national-labs-empowerment@us-rse.org">national lab empowerment working group list</a>. 
```

Reply:
adding line breaks to make changes easier to see; also adding an Oxford comma

Reply:
It would potentially be good to say what Slack channel, and maybe link to it.

Reply:
> It would potentially be good to say what Slack channel, and maybe link to it.

I thought we may change the name of the slack channel because it was so long, but I guess I will add it for now.